### PR TITLE
Improve Dashboard Site Homepage

### PR DESCRIPTION
**Pull request by Nikhil-Kumar1**
Date:  07 Sep 2024

Developer Name:  Nikhil Kumar

## Description

1.  Improved styling of the Dashboard page and added description.
2.  Improved the hover effect and the colour.

Screenshots

Image after changes-
image

Image before changes-
image

### CODE REVIEW

**Review by iamitprakash — CHANGES_REQUESTED** [review]
Details are missing

File: style.css [hunk 1]
```diff
@@ -236,11 +244,25 @@ footer {
...
 
-@media only screen and (max-width: 600px) {
+/* @media only screen and (max-width: 600px) {
+  .buttonSection {
+    flex-direction: column;
+  }
+  .action-button {
+    text-align: center;
+  }
+} */
```
Comment (pallabez) [inline style.css:254]: Can we remove this if we are not using this?

**Comment by pallabez** [comment]
The cards font weight looks too strong. Can we take the reference from /groups?
image


```
<div class="card">
        <h5 class="card__title">Too Much Truth</h5>
        <p class="card__description">Please ask the role creator or admin to update this group description.</p>
</div>
    
.card {
     background: var(--color-group-background);
     padding: 20px;
     border-radius: 15px;
     border: 1px solid hsla(0, 0%, 92%, 1);
     box-shadow: 1px 2px 4px 0px hsla(0, 0%, 0%, 0.1);
}

.card__title {
  font-size: 1.1rem;
  font-weight: 600;
}

.card__description {
  margin-top: 0.6rem;
  color: var(--color-text-secondary);
  font-weight: 400;
  font-size: 0.8rem;
  line-height: 18px;
}
```

**Comment by pallabez** [comment]
Thanks for changing the font weight. Can we do the same for box shadow and font color? Also would recommend to remove the hover effect. What do you think?

**Comment by Nikhil-Kumar1** [comment]
Yes, I will do it

**Comment by pallabez** [comment]
Discussed 

```
      <a
        class="card"
        href="/goal/index.html"
        id="create-goal"
      >
        <h5 class="card__title">Create Goals</h5>
        <p class="card__description">
          Set and track goals to align efforts and measure progress.
        </p>
      </a>
```

```
.card {
  margin: 0;
  background-color: white;
  text-decoration: none;
  padding: 20px;
  width: 300px;
  border-radius: 15px;
  border: 1px solid hsla(0, 0%, 92%, 1);
  box-shadow: 1px 2px 4px 0px hsla(0, 0%, 0%, 0.1);
}

.card__title {
  font-size: 1.1rem;
  font-weight: 600;
  color: hsla(0, 0%, 21%, 1);
}

.card__description {
  margin-top: 0.6rem;
  color: hsla(0, 0%, 21%, 0.75);
  font-weight: 400;
  font-size: 0.8rem;
  line-height: 18px;
}

```

**Review by pallabez — APPROVED** [review]

**Comment by Nikhil-Kumar1** [comment]
> Details are missing

previously I had written commit01,02,03, but  after that I have started mentioning the details, could you please check once.

**Pull request merged**